packages/zosfiles/__tests__/__unit__/methods/create/Create.unit.test.ts

```diff
@@ -823,14 +823,11 @@ describe("Create data set", () => {
             );
         });
 
-        it("should be able to create a partinioned data set without printing the attributes", async () => {
+        it("should be able to create a partitioned data set without printing the attributes", async () => {
             const response = await Create.dataSet(
                 dummySession,
                 CreateDataSetTypeEnum.DATA_SET_PARTITIONED,
-                dataSetName,
-                {
-                    showAttributes: false
-                } as any
```

Comment:
Curious why this was removed? I think we could retain it along with the other references to `showAttributes` in this file.